### PR TITLE
[MAT-2083] MedicationAdministration Attributes

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
Adding MedicationAdministration attributes: dosage.route, reasonCode, status, and statusReason. 
Adding associated value sets and tests.

Screen Shot 2020-10-20 at 5 04 25 PM

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: [MAT-2083](https://jira.cms.gov/browse/MAT-2083)
- [ ] JIRA ticket links to this PR
- [ ] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [ ] Test fixtures updated and documented as necessary ( see [internal wiki](https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) )
- [ ] Code coverage has not gone down and all code touched or added is covered. 
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here: 
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here: 
- [ ] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [ ] JIRA test links:
- [ ] Justification for using JIRA tests:
- [ ] JIRA tests have been added to sprint

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [ ] JIRA tests have been run and pass
- [ ] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [ ] JIRA tests have been run and pass
- [ ] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

File: app/assets/javascripts/helpers/fhir_value_sets.js.coffee [hunk 1]
```diff
@@ -1006,3 +1003,246 @@
...
+      }]
+    }],
+    "description" : "This value set includes all Route codes from SNOMED CT - provided as an exemplar.",
+    "copyright" : "This value set includes content from SNOMED CT, which is copyright © 2002 International Health Terminology Standards Development Organisation (IHTSDO), and distributed by agreement between IHTSDO and HL7. Implementer use of SNOMED CT is not covered by this agreement.",
+    "compose" : {
+      "include" : [{
+        "system" : "http://snomed.info/sct",
+        "concept" : [
+          {
+            "code" : "284009009",
```
Comment (ghost) [inline app/assets/javascripts/helpers/fhir_value_sets.js.coffee]: It's probably a parent code which has a lot more codes under it

**Review by ghost — APPROVED** [review]

File: app/assets/javascripts/helpers/fhir_value_sets.js.coffee (hunk 1 above)
Comment (jkotanchik-SB) [reply app/assets/javascripts/helpers/fhir_value_sets.js.coffee]: Thanks! Following your advise, it has been updated with the children codes.

**Review by adongare — APPROVED** [review]

File: app/assets/javascripts/helpers/fhir_value_sets.js.coffee [hunk 2]
```diff
@@ -1006,3 +1003,846 @@
...
+    "description" : "This value set includes all Route codes from SNOMED CT - provided as an exemplar.",
+    "copyright" : "This value set includes content from SNOMED CT, which is copyright © 2002 International Health Terminology Standards Development Organisation (IHTSDO), and distributed by agreement between IHTSDO and HL7. Implementer use of SNOMED CT is not covered by this agreement.",
+    "compose" : {
+      "include" : [{
+        "system" : "http://snomed.info/sct",
+        "concept" : [
+          {
+            "code": "Code",
+            "display": "Display"
+          },
```
Comment (adongare) [inline app/assets/javascripts/helpers/fhir_value_sets.js.coffee]: @jkotanchik-SB is this needed?

Comment (ghost) [reply app/assets/javascripts/helpers/fhir_value_sets.js.coffee]: it's an artifact from the web scraper script

Comment (jkotanchik-SB) [reply app/assets/javascripts/helpers/fhir_value_sets.js.coffee]: Thanks for catching it. removed.

**Pull request merged**